examples/wearable/UIComponents/contents/navigationelements/pageindicator/index.html

```diff
@@ -7,9 +7,9 @@
 		Wearable UI
 	</title>
 	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
-	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
```

Comment:
Leave the  `media="all and (-tizen-geometric-shape: circle)"`

Reply:
ok, I left it, but we need to discuss this globally